src/python/grpcio/grpc/_adapter/_low.py

```diff
@@ -170,10 +170,14 @@ def set_credentials(self, creds):
 class Channel(_types.Channel):
 
   def __init__(self, target, args, creds=None):
-    args = list(args) + [
-        (cygrpc.ChannelArgKey.primary_user_agent_string, _USER_AGENT)]
+    args = dict(args)
```

Comment:
@ctiller: is it guaranteed that duplicated channel argument keys will never be useful (e.g. channel arguments are never a multimap)?